src/components/projects/filterbuttons.jsx

```diff
@@ -0,0 +1,127 @@
+"use client";
+import { useState } from "react";
+
+import { IoIosArrowDown, IoIosArrowUp } from "react-icons/io";
+
+function Support({ list, checkedcolor }) {
+  return (
+    <div className="border-2 rounded-md border-gray-500 mt-1">
+      {list.map(function (data) {
+        return (
+          <div className="flex flex-row gap-2 m-2" key={0}>
+            <input
+              type="checkbox"
+              className={`relative peer appearance-none w-4 h-4 bg-gray-300 rounded-sm  mt-1 ${checkedcolor}`}
+            />
+
+            {data}
+
+            <svg
+              className="
+                    absolute 
+                    w-4 h-4 mt-1
+                    hidden peer-checked:block pointer-events-none"
+              xmlns="http://www.w3.org/2000/svg"
+              viewBox="0 0 24 24"
+              fill="none"
+              stroke="white"
+              stroke-width="3"
+              stroke-linecap="round"
+              stroke-linejoin="round"
+            >
+              <polyline points="20 6 9 17 4 12"></polyline>
+            </svg>
+          </div>
+        );
+      })}
+    </div>
+  );
+}
+
+const Filterbuttons = () => {
+  const [open, setOpen] = useState(false);
+  const [open1, setOpen1] = useState(false);
```

Comment:
give them more descriptive labels, makes it easier for others to figure out what it's for
same for the handleOpen names